test/src/org/exist/storage/DirtyShutdownTest.java

```diff
@@ -63,7 +64,7 @@ public void storeRepeatedly() {
             for (int i = 0; i < 50; i++) {
                 try(final Txn transaction = transact.beginTransaction()) {
 
-                    Path f = Paths.get("samples/shakespeare/macbeth.xml");
+                    final Path f = TestUtils.resolveShakespeareSample("macbeth.xml");
```

Comment:
It could be, but that is not the design I chose. There are many tests that load shakespeare text so I felt it was more convenient to offer a function for it.